jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBStatement.java

```diff
@@ -79,18 +79,31 @@
    */
   IoTDBStatement(IoTDBConnection connection, TSIService.Iface client,
       TS_SessionHandle sessionHandle,
-      int fetchSize, ZoneId zoneId) {
+      int fetchSize, ZoneId zoneId) throws SQLException {
     this.connection = connection;
     this.client = client;
     this.sessionHandle = sessionHandle;
     this.fetchSize = fetchSize;
     this.batchSQLList = new ArrayList<>();
     this.zoneId = zoneId;
+    requestStmtId();
+  }
+
+  // only for test
```

Comment:
I think these annotations can be extracted to one single module, or if I want to use the @TestOnly annotation, I have to add the server dependency in the pom.xml, but actually, I only use that annotation in that jar. It will cause the jdbc module's with-dependency-jar to be very large.

Reply:
I see.